### PR TITLE
Operations / github action / remove old ones

### PR DESCRIPTION
**Pull request by manolisliolios**

### CODE REVIEW

File: scripts/utils/utils.ts [hunk 1]
```diff
@@ -153,8 +153,6 @@ export const prepareMultisigTx = async (
 
 	// Prevent any possible RGP changes across epoch change, which would invalidate the transaction.
 	tx.setGasPrice(1_000);
-	// set the gas budget.
-	tx.setGasBudget(2_000_000_000);
```
Comment (jnaulty) [inline scripts/utils/utils.ts:157]: this isn't needed anymore?

Comment (manolisliolios) [reply scripts/utils/utils.ts:157]: Yep, SDK takes care of this

File: scripts/transactions/mass-ops.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+	const names = reservedObjects
+		.filter((obj) => namesToTransfer.includes(obj.data.content.fields.domain_name))
+		.map((x) => x.data.objectId);
+	// transfer the list.
+	tx.transferObjects(
+		names,
+		tx.pure.address('0x11469060268ba1d611e3ad95f3134332f68e21198ce068a14f30975336be9ca1'),
+	);
+
+	// transfer `notifi.sui` to `0x2a25e5d858849bf2af0bf30aaa106bff8cdce25b9ae8ec3acfe1f2c346f30c36`
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:73]: ✅

File: scripts/transactions/mass-ops.ts [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+const run = async () => {
+	const config = mainPackage.mainnet;
+
+	// This is a list of free names and their addresses.
+	const freeNamesAddresses = [
+		...new Set([
+			'0x29aa83013e8e5646f479188761e510fd4e7062123d0e1f0ffc654ecfde4d7550',
+			'0x74bd991f694eb5792c10c3715a125629da67a0494e95b299ae23264210a53cf5',
+			'0x29aa83013e8e5646f479188761e510fd4e7062123d0e1f0ffc654ecfde4d7550',
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:33]: this is listed twice in the request and here (didn't notice until i started verifying addresses.)

Comment (jnaulty) [reply scripts/transactions/mass-ops.ts:33]: `0x74bd991f694eb5792c10c3715a125629da67a0494e95b299ae23264210a53cf5`

File: scripts/transactions/mass-ops.ts [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+	}
+
+	return result;
+}
+
+const run = async () => {
+	const config = mainPackage.mainnet;
+
+	// This is a list of free names and their addresses.
+	const freeNamesAddresses = [
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:29]: these are the same as addresses requested (did notice a duplicate)

File: scripts/transactions/mass-ops.ts [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+	// Transfer list of names to the specified wallet
+	const namesToTransfer = ['gateio.sui', 'bybit.sui', 'okx.sui', 'binance.sui', 'kucoin.sui'];
+
+	const names = reservedObjects
+		.filter((obj) => namesToTransfer.includes(obj.data.content.fields.domain_name))
+		.map((x) => x.data.objectId);
+	// transfer the list.
+	tx.transferObjects(
+		names,
+		tx.pure.address('0x11469060268ba1d611e3ad95f3134332f68e21198ce068a14f30975336be9ca1'),
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:70]: confirmed 0x11469060268ba1d611e3ad95f3134332f68e21198ce068a14f30975336be9ca1 is the target address for receiving

File: scripts/transactions/mass-ops.ts [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+		]),
+	];
+
+	const tx = new TransactionBlock();
+
+	// remove discount for NS objects
+	removeDiscountForType(tx, config, `${config.packageId}::suins_registration::SuinsRegistration`);
+
+	// Transfer list of names to the specified wallet
+	const namesToTransfer = ['gateio.sui', 'bybit.sui', 'okx.sui', 'binance.sui', 'kucoin.sui'];
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:62]: confirmed these addresses are requested for transfer

**Review by jnaulty — APPROVED** [review]
looks good!
one minor note on a duplicate address

File: scripts/transactions/mass-ops.ts [hunk 7]
```diff
@@ -77,6 +77,15 @@ const run = async () => {
 		tx.pure.address('0x2a25e5d858849bf2af0bf30aaa106bff8cdce25b9ae8ec3acfe1f2c346f30c36'),
 	);
 
+	// transfer `stashed.sui` to `0x5a3afb4e2d6421488d4417f8cbdaf276079dd6f9c0195d8c8453c7a56d863194`
+	const stashedObj = reservedObjects.find(
+		(obj) => obj.data.content.fields.domain_name === 'stashed.sui',
+	);
+	tx.transferObjects(
+		[stashedObj!.data.objectId],
+		tx.pure.address('0x5a3afb4e2d6421488d4417f8cbdaf276079dd6f9c0195d8c8453c7a56d863194'),
```
Comment (jnaulty) [inline scripts/transactions/mass-ops.ts:86]: recipient address matches Sean's request

**Review by jnaulty — APPROVED** [review]

**Pull request merged**